com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -1182,6 +1182,45 @@ def __init__(self, rest_client, credentials):
     def _get_url(self, req_name):
         return self._credentials['rest_url'] + self._credentials[req_name]
 
+    def submit_job(self, sab_file, application_name,  parameters=[], configuration={}, tracing='info' ):
+        """Submit a compiled Streams Application Bundle (a SAB file) to run in this Streamin Analytics service.
+        
+        Args:
+        sab_file (string): path to the compiled SAB file containing the application to be submitted
+        application_name (string): name of the application in the SAB file
```

Comment:
Not sure why we need to know the application name.

Reply:
We don't, so I've removed it.